src/google/protobuf/port_def.inc

```diff
@@ -257,6 +257,7 @@
 
 #define PROTOBUF_VERSION 3007000
 #define PROTOBUF_MIN_HEADER_VERSION_FOR_PROTOC 3007000
+#define PROTOBUF_MIN_PROTOC_VERSION 3007000
```

Comment:
All the macros in this file are protected by an `#ifdef`/`#error` to ensure that the macro is not already set, and a `#undef` in `src/google/protobuf/port_undef.inc`. Should this new macro have these guards as well?

Reply:
+1